public/slack-app-registry.json

```diff
@@ -13,11 +13,16 @@
                 "description" : "Returns a random xkcd comic.",
                 "manual" : "THERE IS NO DOCUMENTATION WEBSITE. MAKE YOUR OWN."                
             },
-	    "lowPriority" : {
-		"usage" : "/struggle lowPriority",
-		"description" : "Mark has no idea what this function does.",
-		"manual" : "Hope Brett doesn't kill us."
-	    }
+            "lowPriority" : {
+                "usage" : "/struggle lowPriority",
+                "description" : "Mark has no idea what this function does.",
+                "manual" : "Hope Brett doesn't kill us."
+            },
+            "inviteme" : {
```

Comment:
This has to be the name of the function. In this case, it would be `/struggle inviteme`